src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -165,15 +165,12 @@ OverlayImpl::OverlayImpl (
         collector,
         [counts = m_traffic.getCounts(), collector]()
         {
-            std::vector<beast::insight::Gauge> ret;
-            ret.reserve(counts.size() * 4);
+            std::vector<TrafficGauges> ret;
+            ret.reserve(counts.size());
 
-            for (auto const& s : counts)
+            for (size_t i =0; i < counts.size(); ++i)
```

Comment:
nit: spacing/formatting throughout these files. `i = 0` and `counts[i].name, collector` etc. (spaces after commas, spaces in-between operator= operands)

Reply:
Fixed, thanks!